api/README.md

```diff
@@ -0,0 +1,17 @@
+<h1>API</h1>
```

Comment:
It would probably be best to use Markdown for the readme file.  It will be a quick port over!